src/neuroconv/datainterfaces/ophys/scanimage/scanimageimaginginterface.py

```diff
@@ -65,9 +72,16 @@ def get_metadata(self) -> dict:
 
         metadata = super().get_metadata()
 
-        if "state.internal.triggerTimeString" in self.image_metadata:
-            extracted_session_start_time = dateparse(self.image_metadata["state.internal.triggerTimeString"])
```

Comment:
Can you re-add the `dateparse` wrapping around the return below? It's causing certain tests to fail in the full CI

Reply:
Ah, fair point, and that does indeed make my utility function less useful